package.json

```diff
@@ -1,33 +1,52 @@
 {
   "name": "rds-contracts",
   "version": "0.0.1",
-  "description": "contracts for incentivized pinning",
-  "main": "truffle-config.js",
-  "directories": {
-    "test": "test"
+  "description": "Contracts for incentivized pinning",
+  "keywords": [
+    "RIF",
+    "RSK",
+    "pinning"
+  ],
+  "homepage": "https://github.com/rsksmart/rif-marketplace-storage-pinning",
+  "bugs": {
+    "url": "https://github.com/rsksmart/rif-marketplace-storage-pinning/issues"
   },
-  "scripts": {
-    "test": "truffle test"
+  "license": "MIT",
+  "author": {
+    "name": "Rinke Hendriksen",
+    "email": "<rinke@iovlabs.org>"
+  },
+  "files": [
+    "types"
+  ],
+  "directories": {
+    "contracts": "contracts",
```

Comment:
Just FYI `contracts` is not valid, but you can keep it.
https://docs.npmjs.com/files/package.json#directories